definitions/npm/react-redux_v5.x.x/flow_v0.53.x-v0.53.x/react-redux_v5.x.x.js

```diff
@@ -56,9 +56,33 @@ declare module "react-redux" {
     subKey?: string
   ): Provider<*, *>;
 
+  declare function areStatesEqual(
+    nextState?: Object,
+    state?: Object,
+  ): boolean;
+
+  declare function areStatesEqual(
+    nextOwnProps?: Object,
+    ownProps?: Object,
+  ): boolean;
+
+  declare function areStatePropsEqual(
+    nextStateProps?: Object,
+    stateProps?: Object,
+  ): boolean;
+
+  declare function areMergedPropsEqual(
+    nextMergedProps?: Object,
+    mergedProps?: Object,
+  ): boolean;
+
   declare type ConnectOptions = {
     pure?: boolean,
-    withRef?: boolean
+    areStatesEqual?: areStatesEqual,
+    areOwnPropsEqual?: areOwnPropsEqual,
```

Comment:
You have to provide a definition for `areOwnPropsEqual` as you did for `areStatesEqual`.